arabic/dream/It was a nice dream.md

```diff
@@ -0,0 +1,2 @@
+كنت أحاول الفرار من المجرمين وبينما أركض قلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت فواجهتهم حتى قضيت على أربعة منهم وفر الباقى قبل أن ينادينى أحدهم لأسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ  
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.